client/room.html

```diff
@@ -320,7 +320,7 @@ <h1>Save game</h1>
       </div>
       <div>
         <div icon="save">
-          <p>This creates a save state of the currently active game so that you can start a different game and come back to this one later.</p>
+          <p>This creates a save state of the currently active game so that you can start a different game and come back to this one later. Only use this for playing games. For creating or editing, use the "+Add Game" button or sidebar hamburger icon "Edit" button.</p>
```

Comment:
I'd eliminate "Only use this for playing games." I don't think these words really add meaning to the overall message.